libs/event/src/lib/form/shell/shell.component.html

```diff
@@ -32,13 +32,19 @@ <h2>Details</h2>
       </section>
       <event-details-edit [form]="form" showPrivacy></event-details-edit>
     </mat-card>
-
-    <nav mat-tab-nav-bar>
-      <ng-container *ngFor="let tab of (tabs$ |async)">
-        <a mat-tab-link [routerLink]="tab.path" routerLinkActive #rla="routerLinkActive" [active]="rla.isActive">
-          {{ tab.label }}</a>
-      </ng-container>
-    </nav>
+    <div class="container-nav-button">
```

Comment:
```suggestion
    <div>
```

you don't need a class.

this should work:
```

form {
  div {
  display: flex;
  justify-content: space-between;
}
}
```